docs/fast_data/configuration/config_maps/kafka_projection_updates.mdx

```diff
@@ -101,7 +100,30 @@ module.exports = async function* myCustomStrategy ({ logger, dbMongo }, { after
 
 </p>
 </details>
-:::
+
+While we only extracted the `after` property in the examples before, the whole object has actually the following structure:
+
+```typescript
+{
+    operationType: string
+    operationTimestamp: string
+    before?: Document | null
+    after?: Document | null
+    primaryKeys: (string | number | symbol)[]
+    __internal__kafkaInfo: {
+      timestamp: string
+      topic: string
+      key: string
+      partition: number
+      offset: number
+  };
+}
+```
+
+You might notice that it does not look quite like the incoming `pr-update` message you might have,
+this is because we transform the message into a unique format to represent the various versions of the message.
+So if you switch your Real-time Updater (pr-update version 1.0) for a Projection Storer (pr-update version 2.0)
+your manual strategy will still work exactly as it did before.
```

Comment:
```suggestion
:::important
You might notice that it does not look quite like the incoming `pr-update` message you might have,
this is because we transform the message into a unique format to represent the various versions of the message.
So if you switch your Real-time Updater (pr-update version 1.0) for a Projection Storer (pr-update version 2.0)
your manual strategy will still work exactly as it did before.
:::
```